data/movies.json

```diff
@@ -0,0 +1,3 @@
+{
+  "movies": []
+}
```

Comment:
There is no mechanism to add this file dynamically. kindly add a function to add this file.